server/routes/administration/room-locations/room-locations.controller.js

```diff
@@ -0,0 +1,240 @@
+(function() {
+  'use strict';
+  const _ = require('lodash');
+  const { validate } = require('validate.js');
+  const { courtroomsDAO } = require('../../../objects/administration');
+  const validator = require('../../../config/validation/create-courtroom');
+  const { replaceAllObjKeys } = require('../../../lib/mod-utils');
+
+  module.exports.getRooms = function(app) {
+    return (req, res) => {
+      return res.redirect(app.namedRoutes.build('administration.room-locations.location.get', {
+        locationCode: req.session.authentication.locCode,
+      }));
+    };
+  };
```

Comment:
Why use this to redirect to the `getRoomLocations` controller function and not call it directly? Is it just for handling the locationCode?

Reply:
@ricardofreitasrocha Currently i had it for when a court user clicks on the link in the nav bar, i dont think theres a way for us to access the location code directly in nunjucks without first passing it in, and i dont want to have to pass it in everytime we render something that uses court-admin nav.
Open to suggestions though?

Reply:
that works for me, was just a general question 😄